javaslang-test/src/test/java/javaslang/test/GenTest.java

```diff
@@ -44,6 +44,14 @@ public void shouldCreateGenOfSeedAndFunction() {
         assertThat(gen.apply(RANDOM)).isEqualTo(3);
     }
 
+    @Test
+    public void shouldCreateGenOfFixedValues() {
+        final Gen<Integer> gen = Gen.fixed(1,2,3);
```

Comment:
Any suggestions on how I would even _approach_ writing tests for arbitrary values....